[en] Update middleware Kafka-userguide (#4779)
DaoCloud/DaoCloud-docs: docs/en/docs/middleware/kafka/user-guide/insight.md

  - toc
---

---
MTPE: ModetaNiu
Date: 2024-07-03
---

# Instance Insight

> Kafka has built-in Prometheus and Grafana insight modules.

Reviewer: ```suggestion
Kafka has built-in Prometheus and Grafana modules to enhance observability.
```
Insight 是 dce 中的一个产品名。

> 1. On the Message Queue page, click a certain one.

Reviewer: ```suggestion
1. On the Message Queue page, click an intance name.
```
